crates/bin/node/src/logic/inputs.rs

```diff
@@ -81,6 +84,22 @@ pub async fn process_melt_inputs<'a>(
             .await
             .map_err(Error::KeysetCache)?;
 
+        // Validate amount doesn't exceed max_order
+        let max_order = keyset_info.2;
+        let max_value = if max_order >= 64 {
```

Comment:
`max_order` cannot be >= 64 due to checks done in the signer.

I think you can just check `proof.amount > max_order`